tests/unit/factories.cpp

```diff
@@ -20,17 +20,16 @@
 #include <iostream>
 
 #include <graphblas.hpp>
-#include <utils/assertions.hpp>
 #include <graphblas/algorithms/matrix_factory.hpp>
 
 
 using namespace grb;
 using namespace grb::algorithms;
 
-#define ERROR( rc, msg ) {										\
-	std::cerr << "  [Line " << __LINE__ << "] Test FAILED: "	\
-		<< (msg) << std::endl;									\
-	(rc) = FAILED;												\
+#define ERROR( rc, msg ) {                                       \
```

Comment:
:sob:

Reply:
what happened? The alignment for me was off, I guess because it used tabs instead of spaces. But maybe now it doesn't look great in your editor is it?

Reply:
On GitHub it looks messed up but not locally so it's fine